### ISSUE TITLE
Utils guidelines

### ISSUE THREAD

**Issue by itdependsnetworks** [napalm-automation/napalm-utils#4]
In reference to https://github.com/napalm-automation/napalm-base/pull/93

I'm sure all are busy, just wanted to make sure it didn't get lost in shuffle forever.

**Comment by dbarrosop** [comment]
Yeah, this might get some love soon. I am very sorry this was left behind but I was struggling a bit seeing how to put this and other ideas together in a meaningful way but I think I am getting there... 

Again, I am sorry your PR has been sitting idle for a while now, I know how frustrating that can be sometimes : (

**Comment by itdependsnetworks** [comment]
No biggie. Just was on a github follow up tangent and hit this up too :)
